src/main/java/org/openmaptiles/layers/Transportation.java

```diff
@@ -425,6 +536,17 @@ int getMinzoom(Tables.OsmHighwayLinestring element, String highwayClass) {
     return minzoom;
   }
 
+  int getBrunnelMinzoom(Tables.OsmHighwayLinestring element) {
+    try {
+      return Utils.getClippedMinZoomForLength(element.source().length(), 6, 9, 12);
```

Comment:
Does this mean the min zoom that brunnel appears at is 9? Looking at the [OMT diff](https://github.com/openmaptiles/openmaptiles/compare/v3.14..master) and this PR https://github.com/openmaptiles/openmaptiles/pull/1283 it looks like brunnel should appear down to z4 but only on segments with a min length.

Reply:
cc/ @ZeLonewolf who made the change in openmaptiles